xaod_cpp_transformer/requirements.txt

```diff
@@ -10,7 +10,9 @@ pyarrow==0.16.0
 pytest==4.6.11
 pytest-mock==2.0.0
 requests==2.24.0
-servicex-transformer==1.0.3
-uproot==3.11.7
+# servicex-transformer==1.0.3
+# uproot==3.11.7
```

Comment:
Please don't comment out code. We don't need uproot in this transformer, so just delete the requirement

Reply:
agree